### PR TITLE
Creating a padding for ios app

### LINKED ISSUE #52: Change ios status bar type
Status bar shouldn't overlay app content.

### PR DESCRIPTION
**Pull request by arcturus**
@wilsonpage decided to hide the status bar for both android and ios. Do you agree?

Fixes #52

### CODE REVIEW

**Comment by wilsonpage** [comment]
I haven't flashed this patch yet, but I didn't think the intention was to hide the status bar. That kind of feature is usually reserved for immersive media experiences like video or games. I though we just want to make sure the ios status bar occupied space instead of floating on top of the app content.

If this isn't possible we could add some padding to the top of the app in IOS only.

**Comment by arcturus** [comment]
Not hiding the status bar any more. Just added a padding.
Had to split the index into two as it's getting complicated reusing.

File: index.android.js [hunk 1]
```diff
@@ -1,22 +1,16 @@
...
   Component,
   UIManager,
-  Platform
+  AppRegistry
 } = React;
 
 class App extends Component {
   render() {
-    return <ListView style={styles.list}/>;
+    return  <ListView style={styles.list}/>;
```
Comment (wilsonpage) [inline index.android.js:13]: Remove extra whitespace

**Comment by wilsonpage** [comment]
@samgiles we've got a Travis failure:

`https://:@ci.tengam.org/keystore/magnet-release.keystore: Invalid user name.`

I don't think it's related to this patch, so going ahead and landing any way.

**Pull request merged**